src/e2e-test/java/io/cdap/plugin/pubsub/locators/PubSubLocators.java

```diff
@@ -64,6 +64,20 @@ public class PubSubLocators {
 
   @FindBy(how = How.XPATH, using = "//input[@data-cy='numberOfReaders']")
   public static WebElement numberOfReaders;
+  @FindBy(how = How.XPATH, using = "//input[@placeholder='Field name' and @value='']")
```

Comment:
If the locator doesn't have a data-cy or data-test-id  try writing the locator which won't break. (Also no need to include value here as an and)

Reply:
this is the complete locator as we are adding the fields in which this value section is empty.